### DISCUSSION TITLE
Sum of two resulting tables

### DISCUSSION THREAD

**Discussion by kaasym** [Universite-Gustave-Eiffel/NoiseModelling#601] [Ideas]
Hi, I have a request. Would it be possible to improve the program by the sum of two resulting tables? And the same for isolins? I need to add two results, one for line noise sources and the other for point noise sources. For a few points I can add it in Excel, but I can't do a bigger area. Especially for isolins. Thank you. Jirka

**Comment by pierromond** [comment]
We could do a WPS Script for that... I add it as an Issue, when we will find some time, we could do it ! Thank you for the suggestion.

**Reply by kaasym** [reply]
Thank you so much.
